[1.19.2] Updated Oculus & Rubidium to fix #128

F0x06 opened the pull request:
Hi this PR fixes #128 hope this helps,
best regards,
F0x

## List of changes
Bumped version to `1.3.6`
To fix the build/run:
  - Switched Aether to curse maven
  - Added Curios as dependency for Aether

Updated Oculus to `Oculus mc1.19.2-1.6.9`
Updated Rubidium to `Rubidium mc1.19.2-0.6.2c`

Fixed Sodium Mixin `getTerrainVertexShaderSource` -> `getTerrainSolidVertexShaderSource` 
Fixed Iris Mixin `init`
Fixed Iris Mixin `getCurrentDimension`

PS: Attached a zipped .jar for people to test/use
[immersive-portals-2.3.6-mc1.19.2-forge.zip](https://github.com/iPortalTeam/ImmersivePortalsModForForge/files/13400316/immersive-portals-2.3.6-mc1.19.2-forge.zip)

F0x06 committed:
To fix the build/run:
  - Switched Aether to curse maven
  - Added Curios for Aether

Updated Oculus to `Oculus mc1.19.2-1.6.9`
Updated Rubidium to `Rubidium mc1.19.2-0.6.2c`

Fixed Sodium Mixin `getTerrainVertexShaderSource` -> `getTerrainSolidVertexShaderSource`
Fixed Iris Mixin `init`
Fixed Iris Mixin `getCurrentDimension`

F0x06 commented:
Noticed afterwards may be a duplicate of 

https://github.com/iPortalTeam/ImmersivePortalsModForForge/pull/140
https://github.com/iPortalTeam/ImmersivePortalsModForForge/pull/137

Changes merged inside this PR

F0x06 committed:
Merged fixes from pull requests #140 and #137

Nick1st commented on build.gradle:
```diff
@@ -53,14 +53,6 @@ allprojects {
         maven {
             url = "https://modmaven.dev/"
         }
-        maven {
-            name "aether"
-            url = "https://maven.pkg.github.com/Gilded-Games/The-Aether"
-            credentials {
-                username "$mavenDownUser"
-                password "$mavenDownPassword"
-            }
-        }
```
Does Gilded Games no longer publish artifacts to this this maven?

F0x06 replied:
i don't think so, i just switched for simplicity and to avoid username/password, i'll rollback changes

F0x06 replied:
After further investigation it seems the project has moved to https://maven.pkg.github.com/The-Aether-Team/The-Aether (https://github.com/The-Aether-Team/The-Aether) adapted to use github maven

F0x06 replied:
However on the latest version of Aether the portal is not an Immersive Portal, is this caused by the upgrade?

Nick1st replied:
No, I don't think so. IIrc the datapack loading stuff is broken, as such those custom portals aren't working. (You'd also need a datapack enabling compatibility)

Nick1st commented on imm_ptl_core/src/main/java/qouteall/imm_ptl/core/compat/mixin/MixinIrisIris.java:
```diff
@@ -2,6 +2,7 @@
 
 import net.coderbot.iris.Iris;
 import net.coderbot.iris.shaderpack.DimensionId;
```
If this import is no longer required it should be removed.

F0x06 replied:
All imports on this file are still in use. `cir.setReturnValue(DimensionId.OVERWORLD)` and `@Mixin(value = Iris.class, remap = false)`

Nick1st replied:
Oh, I missed that setReturnValue. Currently without an IDE :sweat_smile: That's fine then :+1:

Nick1st commented on imm_ptl_core/src/main/java/qouteall/imm_ptl/core/compat/mixin/MixinSodiumChunkShaderInterface.java:
```diff
@@ -16,53 +16,56 @@
-        at = @At("RETURN"),
-        require = 0,
-        remap = false
+            method = "<init>",
+            at = @At("RETURN"),
+            require = 0,
+            remap = false
```
Can you please remove the added spaces please?

Nick1st commented on imm_ptl_core/src/main/java/qouteall/imm_ptl/core/compat/mixin/MixinSodiumChunkShaderInterface.java:
```diff
@@ -16,53 +16,56 @@
-        ShaderBindingContext context,
-        ChunkShaderOptions options,
-        CallbackInfo ci
+            ShaderBindingContext context,
+            ChunkShaderOptions options,
+            CallbackInfo ci
```
Same here

Nick1st commented on imm_ptl_core/src/main/java/qouteall/imm_ptl/core/compat/mixin/MixinSodiumChunkShaderInterface.java:
```diff
@@ -16,53 +16,56 @@
-        method = "setup",
-        at = @At("RETURN"),
-        remap = false
+            method = "setup",
+            at = @At("RETURN"),
+            remap = false
```
And here

Nick1st commented on imm_ptl_core/src/main/java/qouteall/imm_ptl/core/compat/mixin/MixinSodiumChunkShaderInterface.java:
```diff
@@ -16,53 +16,56 @@
-                    (float) equation[1],
-                    (float) equation[2],
-                    (float) equation[3]
+                        uIPClippingEquation,
+                        (float) equation[0],
+                        (float) equation[1],
+                        (float) equation[2],
+                        (float) equation[3]
```
And here

Nick1st commented on imm_ptl_core/src/main/java/qouteall/imm_ptl/core/compat/mixin/MixinSodiumChunkShaderInterface.java:
```diff
@@ -16,53 +16,56 @@
                 GL20C.glUniform4f(
-                    uIPClippingEquation,
-                    0, 0, 0, 1
+                        uIPClippingEquation,
+                        0, 0, 0, 1
```
And last time here.

Nick1st reviewed (changes requested):
Hey, first of all thank you for your contribution. I'm looking forward to merge this, can you please fix those minor issues I marked.
Thanks.

F0x06 committed:
Made fixes requested by the autor.

Switch to the new aether repo (https://maven.pkg.github.com/The-Aether-Team/The-Aether)

Nick1st reviewed (approved):
I'll try to get this merged shortly
